api/controllers/itemController.js

```diff
@@ -0,0 +1,27 @@
+const { itemService } = require('../services')
+const { asyncWrap } = require('../utils/error')
+
+// for a test
+const getAllItems = async (req, res) => {
+  const items = await itemService.getAllItems();
+  return res.status(200).json({ data : items });
+};
+
+const getItemById = async (req, res) => {
+  try {
+    const itemId = req.params.itemId;
+    if (!itemId) {
+      return res.status(400).json({ message : 'KEY_ERROR' });
+    }
+    const itemById = await itemService.getItemById(itemId);
+    return res.status(201).json({ data : itemById });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message : err.message });
+  }
+}
```

Comment:
- if같이 새로운 구문이 시작 되면 공백을 주어서 가독성을 높여주면 좋을 것 같습니다.
- itemService.getItemById을 통해서 가지고 온 결과물이 결국 item이므로 변수면을 item으로 바꾸면 더 좋을 것 같아요.
- 상태코드 201은 created를 의미합니다. 200으로 수정해 주세요/

```suggestion
    const itemId = req.params.itemId;
    
    if (!itemId) {
      return res.status(400).json({ message : 'KEY_ERROR' });
    }
   
     const item = await itemService.getItemById(itemId);
    return res.status(200).json({ data : item });
  } catch (err) {
    console.log(err);
    return res.status(err.statusCode || 500).json({ message : err.message });
  }
}
```